### PR TITLE
Fixed error bar plotting.

### PR DESCRIPTION
**Pull request by jpa198**
Corrected im_names debug value.
Fixed python2ism in plot_items
Added spatial epsilon value generation script. Intended to feed into plot_items

### CODE REVIEW

File: visualization/comparisons/spatial_im_rrup_epsilon_plot.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        "--dist_min", default=0.1, type=float, help="GMPE param DistMin, default 0.1 km"
+    )
+    parser.add_argument(
+        "--dist_max",
+        default=100.0,
+        type=float,
+        help="GMPE param DistMax, default 100.0 km",
+    )
+    parser.add_argument(
+        "--n_val", default=51.0, type=float, help="GMPE param n_val, default 51.0"
```
Comment (viktor76525) [inline visualization/comparisons/spatial_im_rrup_epsilon_plot.py]: n_val sounds like it should be an int

File: visualization/comparisons/spatial_im_rrup_epsilon_plot.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    setup_dir(os.path.dirname(args.out_file))
+    return args
+
+
+def get_print_name(im, comp):
+    """Takes in an im and component and creates a printable name from them
+    In the case of pSA ims the period is processed such that the letter p is used in place of a decimal point and any
+    trailing 0s are trimmed.
+    pSA_0.02 -> pSA(0p02)
+    pSA_0.5 -> pSA(0.5)
```
Comment (shot0829) [inline visualization/comparisons/spatial_im_rrup_epsilon_plot.py]: base on the code this should be pSA(0p5)?

Comment (jasonmotha) [reply visualization/comparisons/spatial_im_rrup_epsilon_plot.py]: if this is the name used on the plot then the output should look like pSA(0.5s)

Comment (jasonmotha) [reply visualization/comparisons/spatial_im_rrup_epsilon_plot.py]: But it doesn't quite look like the comments show how it should be.

Comment (jasonmotha) [reply visualization/comparisons/spatial_im_rrup_epsilon_plot.py]: Internal use (aka not on plots, but in csv files) should be "pSA_0.5"

**Pull request merged**